storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/schema/SchemaFinalized.java

```diff
@@ -31,4 +34,35 @@
   KvStoreColumn<Bytes32, SignedBeaconBlock> getColumnNonCanonicalBlocksByRoot();
 
   KvStoreColumn<UInt64, Set<Bytes32>> getColumnNonCanonicalRootsBySlot();
+
+  @Override
+  default List<KvStoreColumn<?, ?>> getAllColumns() {
+    return List.of(
+        getColumnSlotsByFinalizedRoot(),
+        getColumnFinalizedBlocksBySlot(),
+        getColumnFinalizedStatesBySlot(),
+        getColumnSlotsByFinalizedStateRoot(),
+        getColumnNonCanonicalBlocksByRoot(),
+        getColumnNonCanonicalRootsBySlot());
+  }
+
+  default Map<String, KvStoreColumn<?, ?>> getColumnMap() {
+    final List<KvStoreColumn<?, ?>> allColumns = getAllColumns();
+    return Map.of(
+        "SLOTS_BY_FINALIZED_ROOT", allColumns.get(0),
+        "FINALIZED_BLOCKS_BY_SLOT", allColumns.get(1),
+        "FINALIZED_STATES_BY_SLOT", allColumns.get(2),
+        "SLOTS_BY_FINALIZED_STATE_ROOT", allColumns.get(3),
+        "NON_CANONICAL_BLOCKS_BY_ROOT", allColumns.get(4),
+        "NON_CANONICAL_BLOCK_ROOTS_BY_SLOT", allColumns.get(5));
+  }
```

Comment:
I think I'd go the other way here - we don't want this method to have a brittle dependency on the order of columns from `getAllColumns()`.  So I'd have this method call the getter methods and then `getAllColumns` is just `return new ArrayList<>(getColumnMap().values())`. Or possibly can be changed to return a `Collection` and then it just returns `getColumnMap().values()` directly.

Reply:
Missed this change as well (easy to loose things in GitHub).